python/paddle/distributed/auto_parallel/api.py

```diff
@@ -1128,6 +1139,20 @@ def __init__(self, optimizer, shard_fn=None, gradient_accumulation_steps=1):
             for param in self._inner_opt._parameter_list:
                 self._shard_fn._shard_parameter(param)
 
+        self.fuse_param_view = []
+        self.param_storage = []
+        self.grad_storage = []
+        self._sharding_group = None
+        self._mp_group = None
+        self.do_tensor_fusion_once = True
+        self._strategy = Strategy()
+        self.enable_tensor_fusion = (
+            os.getenv("FLAGS_enable_tensor_fusion") == '1'
+        )
+        self.enable_sharding_overlap = (
```

Comment:
sharding_overlap will not be set through FLAGS in the future and better to configure it with `Strategy`